fdm-app/app/routes/farm.$b_id_farm.$calendar.norms.tsx

```diff
@@ -108,85 +107,84 @@ export async function loader({ request, params }: LoaderFunctionArgs) {
             }
         })
 
-        // Currently only 2025 is supported
-        if (calendar !== "2025") {
-            return {
-                farm: farm,
-                b_id_farm: b_id_farm,
-                b_id: b_id,
-                calendar: calendar,
-                farmOptions: farmOptions,
-                fieldOptions: fieldOptions,
-                fieldNorms: undefined,
-                farmNorms: undefined,
+        const asyncData = (async () => {
+            // Currently only 2025 is supported
+            if (calendar !== "2025") {
+                return {}
             }
-        }
 
-        let fieldNorms = undefined as
-            | {
-                  b_id: string
-                  b_area: number
-                  norms: {
-                      manure: GebruiksnormResult
-                      phosphate: GebruiksnormResult
-                      nitrogen: GebruiksnormResult
-                  }
-              }[]
-            | undefined
-        let farmNorms = undefined as AggregatedNormsToFarmLevel | undefined
-        let errorMessage = null as string | null
-        try {
-            // Calculate norms per field
-            const functionsForms = createFunctionsForNorms("NL", calendar)
+            let fieldNorms = undefined as
```

Comment:
Please do not reformat the code, unless it was incorrectly formatted earlier. Use Biome to format